protobuf/v1/schema.proto

```diff
@@ -0,0 +1,272 @@
+syntax = "proto3";
+
+package schema.v1;
+
+option go_package = "schema/service/v1";
+
+import "google/protobuf/struct.proto";
+import "google/api/annotations.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+message ResolveBooleanRequest {
+    string flag_key = 1;
+    google.protobuf.Struct context = 2;
+    bool default_value = 3;
+}
+message ResolveBooleanResponse {
+    bool value = 1;
+    string reason = 2;
+    string variant = 3;
+}
+
+message ResolveStringRequest {
+    string flag_key = 1;
+    google.protobuf.Struct context = 2;
+    string default_value = 3;
+}
+message ResolveStringResponse {
+    string value = 1;
+    string reason = 2;
+    string variant = 3;
+}
+
+message ResolveNumberRequest {
+    string flag_key = 1;
+    google.protobuf.Struct context = 2;
+    float default_value = 3;
+}
+message ResolveNumberResponse {
+    float value = 1;
+    string reason = 2;
+    string variant = 3;
+}
+
+message ResolveObjectRequest {
+    string flag_key = 1;
+    google.protobuf.Struct context = 2;
+    google.protobuf.Struct default_value = 3;
+}
+message ResolveObjectResponse {
+    google.protobuf.Struct value = 1;
+    string reason = 2;
+    string variant = 3;
+}
+
+service Service {
+    rpc ResolveBoolean(ResolveBooleanRequest) returns (ResolveBooleanResponse) {
+        option (google.api.http) = {
+            post: "/flags/{flag_key}/resolve/boolean"
+            body: "*"
+        };
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Resolve boolean value for flagKey";
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK"
+                    schema: {
+                        example: '{"flag_key":true,"reason":"","variant":""}';
```

Comment:
Should `flag_key` actually be value in this example?

Reply:
these have been removed now, originally these were for padding out the opapiv2 docs, this will no longer be the case